didpass-demo/issuer-demo/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "issuer-demo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@didpass/issuer-sdk": "^0.1.0-beta.2",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.14.16",
+    "@mui/material": "^5.14.17",
+    "http-status-codes": "^2.3.0",
+    "inversify": "^6.0.2",
+    "ioredis": "^5.3.2",
+    "js-cookie": "^3.0.5",
+    "jsonwebtoken": "^9.0.2",
+    "next": "14.0.1",
+    "react": "^18",
+    "react-dom": "^18",
+    "react-qrcode-logo": "^2.9.0",
+    "reflect-metadata": "^0.1.13",
+    "uuid": "^9.0.1"
+  },
+  "devDependencies": {
+    "@types/js-cookie": "^3.0.6",
+    "@types/jsonwebtoken": "^9.0.5",
+    "@types/node": "^20",
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "@types/uuid": "^9.0.7",
+    "i": "^0.3.7",
```

Comment:
lib ini mmg dipakai kah?

Reply:
not intended, removed. kayanya ga sengaja ke add ko wkwkwkwk my bad.